src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
```

Comment:
```suggestion
  return {
```
You can directly return the object without creating this variable